src/platforms/common/blackpill-f4/blackpill-f4.h

```diff
@@ -115,6 +115,41 @@
 #define SWDIO_PORT TMS_PORT
 #define SWDIO_PIN  TMS_PIN
 
+#if 0
+#define TMS_SET_MODE()     gpio_mode_setup(TMS_PORT, GPIO_MODE_OUTPUT, GPIO_PUPD_NONE, TMS_PIN);
+#define SWDIO_MODE_FLOAT() gpio_mode_setup(SWDIO_PORT, GPIO_MODE_INPUT, GPIO_PUPD_NONE, SWDIO_PIN);
+
+#define SWDIO_MODE_DRIVE() gpio_mode_setup(SWDIO_PORT, GPIO_MODE_OUTPUT, GPIO_PUPD_NONE, SWDIO_PIN);
+
+#else
+
+#define SWDIO_MODE_REG_MULT_PB9 (1 << (9 << 1))
+#define SWDIO_MODE_REG_MULT_PB8 (1 << (8 << 1))
```

Comment:
These all need their `U` suffixes to put them in unsigned mode so we don't run the risk of an accidental conversion oops.